### PR TITLE
Cu 86b14rync setup pages UI

### PR DESCRIPTION
**Pull request by Victorportus**
Setup UI and Flow completed

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b14rync Setup Pages UI](https://app.clickup.com/t/86b14rync)

**Review by whamester — APPROVED** [review]

File: src/app/setup/set-up-goal.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                style={{
+                  top: -10,
+                }}
+              >
+                <Image name="level-up-image" />
+              </Stack>
+            </>
+          )}
+        </Center>
+        <ContentCard title={steps[step].title} text={steps[step].text} />
```
Comment (ReinhardtBotha) [inline src/app/setup/set-up-goal.tsx:102]: there are only 2 steps but the step indicator shows 3

File: src/app/setup/enable-notifications.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          }
+        />
+        {/* TODO: Allow notifications functionality */}
+        <Stack w={width * 0.9} gap={12} style={styles.buttonStack}>
+          <Button
+            title="Allow Notifications"
+            onPress={next}
+            variant="primary"
+          />
+          <Button title="Maybe Later" onPress={next} variant="secondary" />
```
Comment (ReinhardtBotha) [inline src/app/setup/enable-notifications.tsx:56]: when you click on Maybe Later it should show this confirmation modal: 
https://www.figma.com/design/PPygQ4sRjPUHr2hBe7aHxe/Updated-Mockups?node-id=4-3815&m=dev
image

**Review by ReinhardtBotha — APPROVED** [review]
Looks great!! 👌

File: src/app/setup/enable-notifications.tsx (hunk 2 above)
Comment (whamester) [reply src/app/setup/enable-notifications.tsx:56]: let's make this change on different PR

**Review by zeckkimon — APPROVED** [review]

**Pull request merged**